backend/apps/readings/models.py

```diff
@@ -33,6 +33,17 @@ def __str__(self):
         """ string representation of this class """
         return f'Document: {self.title} by {self.author}'
 
+    def total_word_count(self):
+        """
+        Computes the total wordcount by iterating through and getting individual wordcounts of
+        all segments
+        :return: int
+        """
+        count = 0
+        for segment in self.segments.get_queryset():
```

Comment:
what you want here is `self.segments.all()` -- get_queryset() will work, but isn't really for use at this level.

(You don't need to know this, but -- `get_queryset()` is a method that you're supposed to override on a model's manager if you want to define fancy behavior for `all()` -- see https://docs.djangoproject.com/en/2.2/topics/db/managers/#modifying-a-manager-s-initial-queryset if you're interested)

Reply:
Ok I fixed this.